pkg/bff/users.go

```diff
@@ -170,8 +171,16 @@ func (s *Server) Login(c *gin.Context) {
 			return
 		}
 
+		// Verify that pending invitations have not expired
+		if err = collaborator.ValidateInvitation(); err != nil {
+			log.Debug().Err(err).Str("email", collaborator.Email).Str("org_id", org.Id).Msg("invalid user invitation")
+			c.JSON(http.StatusForbidden, api.ErrorResponse("user invitation has expired"))
+			return
+		}
+
 		// Other endpoints expect the user's verification status to be up to date
 		collaborator.Verified = *user.EmailVerified
+		collaborator.ExpiresAt = ""
```

Comment:
This is the final part of the invite workflow when the user logs in and the `orgid` is included in the request. At this point, we need to verify that the organization they are trying to log into has their "invitation" record and that the invite has not expired. We will eventually need a way for users to resend invitations, although currently they can accomplish this by deleting and re-inviting the user.